### PR TITLE
Update ore deploy for new additions

### PR DESCRIPTION
**Pull request by Katrix**
Updates the ore deploy for new additions like creating a forum post, and the changelog to add.

Not tested.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ defaultTasks 'clean', 'licenseFormat', 'build'
 apply plugin: 'java'
 
 group = 'org.spongepowered'
-version = '0.8.2-SNAPSHOT'
+version = '0.8.3-SNAPSHOT'
```
Comment (stephan-gh) [inline build.gradle]: Why? 0.8.2 wasn't released yet, so not sure why we'd skip it. (Although the next release will be probably 0.9.0, due to the new ore deploy plugin)

There will be a separate commit to bump the version, though, so please don't do this as part of this PR.

File: src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy [hunk 2]
```diff
@@ -90,13 +92,22 @@ class OreDeployTask extends DefaultTask {
...
-            .addPart('pluginSig', new FileBody(sig.file))
-            .build()
+        MultipartEntityBuilder entityBuilder = MultipartEntityBuilder.create()
+                .addPart('apiKey', new StringBody(apiKey))
+                .addPart('channel', new StringBody(channel))
+                .addPart('recommended', new StringBody(recommended as String))
+                .addPart('pluginFile', new FileBody(plugin.file))
+                .addPart('pluginSig', new FileBody(sig.file))
+
+        if(forumPost != null) {
```
Comment (stephan-gh) [inline src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy]: Can you add a space after `if` here and below?

**Review by stephan-gh — CHANGES_REQUESTED** [review]

File: src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy [hunk 3]
```diff
@@ -51,6 +51,8 @@ class OreDeployTask extends DefaultTask {
     boolean recommended = true
     String channel
     String apiKey
+    Boolean forumPost
```
Comment (stephan-gh) [inline src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy:54]: `boolean` (lower case)

**Review by stephan-gh — CHANGES_REQUESTED** [review]

File: src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy (hunk 3 above)
Comment (Katrix) [reply src/main/groovy/org/spongepowered/gradle/ore/OreDeployTask.groovy:54]: It's intended to be a boxed boolean to differentiate between no value set (null), and some value set.

**Review by stephan-gh — APPROVED** [review]

**Review by kashike — APPROVED** [review]

**Comment by phase** [comment]
Is this only waiting on testing?
